doc/user-guide/jupyter/ipygany.rst

```diff
@@ -15,9 +15,7 @@ for embedded html documentation (as shown here), rapid plotting (as
 compared to the other client jupyterlab plotting modules like
 ``panel`` or ``itkwidgets``).
 
-There is an excellent block post at `ipygany: Jupyter into the third dimension <https://blog.jupyter.org/ipygany-jupyter-into-the-third-dimension-29a97597fc33>`_
```

Comment:
Is the link dead? Or did you remove it for a different reason?

Reply:
The link was dead few days ago but seems to be back now. I'll add again :)

Reply:
Fixed :)